src/sql/first.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE hospital;
+
+CREATE TABLE diseases
+(
+    id          	INT UNSIGNED  AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    nameDiseases    VARCHAR NOT NULL
+);
+
+INSERT INTO diseases (nameDiseases)
+VALUES ('firstdiseases'), ('seconddiseases'), ('thirddiseases');
+
+CREATE TABLE medicines
+(
+    id                      INT UNSIGNED AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    name                    VARCHAR(50) NOT NULL,
+    amount			        INT, NOY NULL,
+    diseasesId              INT UNSIGNED NOT NULL,
+    FOREIGN KEY (diseasesId) REFERENCES diseases(id)
+);
+
+INSERT INTO medicines(name, amount, diseasesId)
+VALUES ('firstMedicinesa', 50, 3), ('secondMedicinesa', 70, 2), ('thirdMedicinesa', 25, 1);
+
+CREATE TABLE patients
+(
+    id                    INT UNSIGNED AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    firstName             VARCHAR(50) NOT NULL,
+    lastName              VARCHAR(50) NOT NULL,
+    gender                CHAR NOT NULL,
```

Comment:
Please avoid direct storage of 'gender'. In this case, it can be just a boolean value. Or you can create a separate 'gender' table and store only 'gender_id' here